assets/src/scss/components/_buttons.scss

```diff
@@ -151,22 +165,30 @@
   }
 }
 
+$donate-button-box-shadow: 0 2px 5px rgba(0, 0, 0, .25);
+
 .btn-donate,
 .wp-block-button.is-style-donate a {
-  background-color: var(--btn-donate-background-color, $aquamarine);
-  color: var(--btn-donate-color, $grey-80);
+  --button-donate-- {
+    background: $aquamarine;
+    color: $grey-80;
+    min-width: 180px;
+    padding: 2px 30px;
+    box-shadow: $donate-button-box-shadow;
+    border-width: 1px;
+    border-color: transparent;
+
+    &:hover,
+    &:focus {
+      background: $aquamarine;
+      color: transparentize($grey-80, 0.2);
+      box-shadow: $donate-button-box-shadow;
+    }
+  }
   line-height: 1.65;
```

Comment:
This line-height should also maybe be a variable?

Reply:
We probably want to look at how to configure button spacing more widely first, which might involve changing the CSS a bit to make it more consistent. Then we probably need to add less variables, or we can add more appropriate / easier to work with ones.